crates/matrix-sdk-ui/src/room_list_service/mod.rs

```diff
@@ -156,6 +156,9 @@ impl RoomListService {
                 .with_to_device_extension(
                     assign!(http::request::ToDevice::default(), { enabled: Some(true) }),
                 );
+        } else {
+            // TODO: This is racy with encryption, needs cross-process lock
```

Comment:
Please update the comment to specify that this is racy if encryption is used within the room list.